digitalpublicgoods/digit-public-finance-management.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "DIGIT Public Finance Management",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://ifix.digit.org/"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
```

Comment:
The current documentation is missing an installation guide to enable a technical person unfamiliar with the project to launch and run the software. Thanks.